test/tools/integration/create-and-destroy-machine.sh

```diff
@@ -7,14 +7,17 @@ cd $(dirname $0)
 export ADDR=$(cat terraform.tfstate |jq -r '.modules[0].resources["hcloud_server.machine-controller-test"].primary.attributes.ipv4_address')
 
 export OS=${1:-ubuntu}
+export TYPE=${2:-hetzner}
 
 ssh_exec() { ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@$ADDR $@; }
 cat <<EOF |ssh_exec
 set -e
 echo "Testing create of a node via machine-controller...."
 ./verify \
-  -input examples/machine-hetzner.yaml \
+  -input examples/machine-$TYPE.yaml \
   -parameters "<< HETZNER_TOKEN >>=$HZ_TOKEN,cri-o=docker,node1=testnode-${CIRCLE_BUILD_NUM:-local},ubuntu=${OS}" \
+  -parameters "<< VSPHERE_PASSWORD >>=${VSPHERE_PASSWORD:-undef},<VSPHERE_USERNAME>=${VSPHERE_USERNAME:-undef}" \
```

Comment:
I think we also need to fix `-parameters`, at the moment the one defined in line 18 is overwritten

Reply:
That should work just fine, check out the changes I made to the verify tool

Reply:
you are right, I missed that one. Nice !